src/azure-cli/azure/cli/command_modules/vm/manual/custom.py

```diff
@@ -54,3 +56,56 @@ def sshkey_create(client,
         logger.warning('Public key is saved to "%s".', public_key_file)
     return client.get(resource_group_name=resource_group_name,
                       ssh_public_key_name=ssh_public_key_name)
+
+
+def sig_shared_gallery_list(client, location, shared_to=None):
+    if shared_to == 'subscription':
+        shared_to = None
+    return client.list(location=location,
+                       shared_to=shared_to)
+
+
+def sig_share_update(cmd, client, resource_group_name, gallery_name, subscription_ids=None, tenant_ids=None,
+                     op_type=None):
+    SharingProfileGroup, SharingUpdate, SharingProfileGroupTypes = cmd.get_models(
+        'SharingProfileGroup', 'SharingUpdate', 'SharingProfileGroupTypes')
+    if subscription_ids is None and tenant_ids is None:
+        raise ValidationError('At least one of subscription ids or tenant ids must be provided')
```

Comment:
Could we use the specific error type `RequiredArgumentMissingError` instead of `ValidationError`?

Reply:
sure